conformance/setup.go

```diff
@@ -129,36 +145,57 @@ func init() {
     }
 }
 `)
-	configContentLength = strconv.Itoa(len(configContent))
-	blobDigest = godigest.FromBytes(configContent).String()
+	configBlobContentLength = strconv.Itoa(len(configBlobContent))
+	configBlobDigest = godigest.FromBytes(configBlobContent).String()
 	if v := os.Getenv(envVarBlobDigest); v != "" {
-		blobDigest = v
+		configBlobDigest = v
+	}
+
+	layerBlobData, err = base64.StdEncoding.DecodeString(layerBase64String)
+	if err != nil {
+		log.Fatal(err)
 	}
 
-	manifestContent = []byte(fmt.Sprintf(
-		"{ \"mediaType\": \"application/vnd.oci.image.manifest.v1+json\", \"config\":  { \"digest\": \"%s\", "+
-			"\"mediaType\": \"application/vnd.oci.image.config.v1+json\","+" \"size\": %s }, \"layers\": [], "+
-			"\"schemaVersion\": 2 }",
-		blobDigest, configContentLength))
+	layerBlobDigest = godigest.FromBytes(layerBlobData).String()
```

Comment:
If we delete all tests with issues... we won't have a way to track them :-) IMO we should lean to adding additional tests and have a config that enables skipping over the tests that are not supported on some registries due to "disagreement" on container standards till we can reach agreement. I believe the empty layer test to be a good test. Thoughts?

Reply:
@mikebrow 
Happy to implement that way. I can return the manifest config to the way it was (empty layer) and only add the layer when a certain environment variable is set.

Reply:
sure or just have two tests one empty and one not and use a ginkgo regex to skip empty for the GCR registry

Reply:
I went with the former, since when I got your response the solution was already underway :-D.

Reply:
by skip empty for GCR I meant let them configure it to skip if that's what they want but we can start with an initial config that maps out what we see passing and what needs to be skipped..